api/query.go

```diff
@@ -122,6 +155,12 @@ func (e *Endpoint) SliceMetadataPost(ctx *gin.Context) {
 	e.sliceMetadata(ctx, *query)
 }
 
+// Slice godoc
+// @Summary  Fetch a single slice in any cube direction
+// @Param    query  query  string  True  "Urlencoded/escaped SliceQuery"
+// @Produce  application/octet-stream
+// @Success  200
+// @Router   /slice  [get]
```

Comment:
Why do I get "example" on post, but not on get? Can I have a "get" example?

Reply:
Examples are deduced from `@Param`'s type. For post that is the SliceQuery, but for Get it's technically a string. Changing `string` to `SliceQuery` makes the docs incorrect/confussing. It's possible to add an examples value after the comment in `@Param`, but that again should reflect the expected type (url encoded string, not SliceQuery)

Reply:
That's not what it expects, the value of query is url encoded, so it looks like:
`query=direction%3D%22inline%22%0Alineno%3D%229961%22%0Avds%3D%22volve%2F32_RLE%22%0Asas%3D%22s...`, which is not a nice example